owasp-dependency-check-suppressions.xml

```diff
@@ -664,4 +664,22 @@
     ]]></notes>
     <cve>CVE-2023-36415</cve>
   </suppress>
+  <suppress>
+    <!-- Covers DOS on identity server by triggering high resource consumption. Used in Azure as a client.
+    Current latest version of Azure BOM (1.2.21) still uses 9.30.2, whereas bug resolved in 9.37.3 -->
```

Comment:
The reasoning is a bit cryptic. Do you mean to say that CVE only impacts the server but not the client?

Reply:
Not clear about that. Simplified the comment to just say that the latest version of BOM still uses vulnerable dependency.